examples/features/authentication/client/main.go

```diff
@@ -16,7 +16,7 @@
  *
  */
 
-// The client demonstrates how to supply an OAuth2 token for every RPC.
+// Client demonstrates how to supply an OAuth2 token for every RPC.
```

Comment:
same: removing `The` doesn't improve much, so let's avoid not so important changes since we are already changing lot of files